Dockerfile

```diff
@@ -30,4 +30,4 @@ RUN yarn install
 
 COPY . .
 
-CMD ["bin/dev"]
+ENTRYPOINT ["docker/entrypoint.sh"]
```

Comment:
Why entrypoint and not cmd? If the former, then we can't override this in docker-compose.yml